### ISSUE TITLE
Project Tutorial Proposal - Soumik Seal

### ISSUE THREAD

**Issue by Soumik8114** [codedex-io/projects#15]
1. Plotting a covid graph using python - Intermediate
2. Creating a telegram bot using python - Intermediate
3. Sudoku Solver using python - Intermediate

**Comment by sonnynomnom** [comment]
Just chatted with the team: Let's do the **Solve Sudoku with Python**!!! 🙌

Can you write the final solution code first before writing the tutorial? cc @Bobliuuu @asiqurrahman 

Also, how did you find us?

**Comment by Soumik8114** [comment]
Yes, I can provide you with the final solution code. I found this repository in https://hacktoberfest-swag.com/ website 😅

**Comment by Soumik8114** [comment]
Here is the code of sudoku solver https://github.com/Soumik8114/Python-Projects/blob/main/sudoku-solver.py

**Comment by Bobliuuu** [comment]
Hey @Soumik8114, great work! Love the comments 🎉 

Just some small changes: 
- Please follow the [formatting guidelines](https://github.com/codedex-io/projects/blob/main/.github/CONTRIBUTING.md). 
- Make sure strings are single quotes
- Python indentation is two spaces
- Some of your comments don't have to span multiple lines, and you're code can be shortened a bit
- `'No solution exists'`
- Grid formatting should be all aligned
- printing -> printGrid
- no space after comma for comments

Once you make those changes, you can start writing your project tutorial! 
Please use the [project template](https://github.com/codedex-io/projects/blob/main/project-template.mdx) provided by Codédex for your tutorial!

**Comment by Soumik8114** [comment]
Okay Thank you for the suggestions :)

**Comment by Bobliuuu** [comment]
No problem!
